linter.py

```diff
@@ -26,14 +27,47 @@ class Rubocop(RubyLinter):
         'ruby on rails',
         'ruby'
     )
-    cmd = 'ruby -S rubocop --format emacs'
+    cmd = None
+    executable = 'ruby'
```

Comment:
If you don't specify `executable` then Sublimelinter doesn't even call the `cmd` method. That wasn't obvious to me, I discovered it through trial and error. However if you read the [docs for this attribute](http://www.sublimelinter.com/en/latest/linter_attributes.html#executable) it does say as much:
> If the name of the executable cannot be determined by the first element of cmd (for example when cmd is a method that dynamically generates the command line arguments), this can be set to the name of the executable used to do listing.

As for why it's `ruby` and not `rubocop` - based on the fact that normally it is automatically determined from the first element of `cmd`, that means it was already using `ruby` so I kept it this way.

Do you think I should add a comment to clarify this?